app/views/users/show.html.erb

```diff
@@ -1 +1,31 @@
-<h1>Here is some details about user</h1>
+<h1>Here is details of Users</h1>
+<section>
+  <div class="d-flex justify-content-between">
+    <div>
+      <img src="<%= @user.photo %>" alt='User Avatar' style="width:20%" />
+    </div>
+    <div class='flex-grow-1 d-flex justify-content-between align-items-center'>
+      <a href="/users/<%= @user.id %>/posts">
+        <h1><%= @user.name %></h1>
+      </a>
+      <p>Number of posts: <%= @user.posts_counter || 0 %></p>
+    </div>
+  </div>
+  <div class='border border-dark p-4'>
+    <h3>Bio</h3>
+    <%= @user.bio %>
+  </div>
+  <div class='posts mt-5'>
+    <% @user.recent_posts.each_with_index do |post,index| %>
+      <div class="border border-dark p-2 mb-2">
+        <h3>post #<%= index+1 %></h3>
+        <p><%= post.text%></p>
+        <div class="d-flex justify-content-end">
+          <span class="">Comments: <%= post.comments_counter || 0 %> </span>
+          <span class="float-right">, Likes: <%= post.likes_counter || 0 %></span>
+        </div>
+      </div>
+    <% end %>
+  </div>
+  <a href="<%= @user.id%>/posts" class="d-flex justify-content-center"> See all posts</a>
+</section>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the user show page is displayed after clicking each user looks like the wireframe design provided.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="457" alt="Screenshot 2023-06-24 at 20 06 59" src="https://github.com/AlaaAlsalem/blogApp/assets/58692590/660904ab-349d-4ed9-accb-08cd08d565fa"> | <img width="486" alt="Screenshot 2023-06-24 at 20 14 50" src="https://github.com/AlaaAlsalem/blogApp/assets/58692590/b97016a9-d03f-4c57-abf4-d0d41385ea1d">